### DISCUSSION TITLE
How to customize metrics?

### DISCUSSION THREAD

**Discussion by JunjieCharles** [open-mmlab/mmediting#1204] [How-to] [answered]
I'd like to add new metrics 'WS-PSNR' and 'WS-SSIM' for evaluation. How to add them?

**Comment by Yshuo-Li** [comment, marked as answer]
Metrics are defined in `https://github.com/open-mmlab/mmediting/blob/master/mmedit/core/evaluation/metrics.py`.

You can add your metrics in `https://github.com/open-mmlab/mmediting/blob/master/mmedit/core/evaluation/metrics.py` or create a new file in  `https://github.com/open-mmlab/mmediting/blob/master/mmedit/core/evaluation/` to add your metrics.

Besides, it will be better if you push your code to this Repo by [PR](https://github.com/open-mmlab/mmediting/pulls). By which we can help you more directly.
